core/DataAccess/LogQueryBuilder.php

```diff
@@ -126,9 +165,29 @@ private function buildWrappedSelectQuery($select, $from, $where, $groupBy, $orde
             $innerGroupBy = false;
         }
 
+        if (!isset($innerGroupBy) && in_array('log_visit', $matchesFrom[1])) {
+            $innerGroupBy = "log_visit.idvisit";
+        } elseif (!isset($innerGroupBy)) {
+            throw new Exception('Cannot use subselect for join as no group by rule is specified');
+        }
+
+        if (!empty($toBeReplaced)) {
```

Comment:
related to above fix when selecting eg twice the `name` column... we then need to make sure to also change orderBy, groupBy etc.